src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Link, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistration } from '../../../api/registrationApi';
+import { Registration, RegistrationStatus } from '../../../types/registration.types';
+
+interface DeletedRegistrationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+  const { t } = useTranslation();
+  const duplicateRegistrationQuery = useQuery({
+    enabled: !!registration.duplicateOf,
+    queryKey: ['registration', registration.duplicateOf],
+    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const duplicateRegistrationTitle =
+    duplicateRegistrationQuery.data?.entityDescription?.mainTitle ?? registration.duplicateOf;
```

Comment:
npår ønsker vi å vise identifier her? blir vel en litt mystisk flashing av verdier, så jeg tenker det kanskje er bedre med en `<Skeleton>` når `duplicateRegistrationQuery.isLoading` 🤔 
```suggestion
    duplicateRegistrationQuery.data?.entityDescription?.mainTitle;
```